tests/integration/targets/snap/aliases

```diff
@@ -11,3 +11,4 @@ skip/freebsd
 skip/osx
 skip/macos
 skip/docker
+disabled
```

Comment:
Why add `disabled`? I'm against disabling tests for this module if possible.

Right now the tests are efficient enough, the one extremely slow one is commented out. As long as the new packages don't make it as extremely slow as before, it should be ok.

Reply:
I could not find a good replacement for that one yet. I just thought the CI pipelines could have a break from that test. If you think that's good enough, I am happy to enable it back.

Reply:
Let's try to remove `disabled` and see how long this one takes.

Also removing that test from CI is generally a bad idea, for example https://github.com/ansible-collections/community.general/pull/8221 is really depending on it running in CI.